EDITORS.md

```diff
@@ -78,3 +79,26 @@ require("lspconfig").ruby_ls.setup({
   end,
 })
 ```
+
+## Sublime Text LSP
+
+[LSP for Sublime Text](https://github.com/sublimelsp/LSP) has built-in support for [Solargraph](https://lsp.sublimetext.io/language_servers/#solargraph) and [Sorbet](https://lsp.sublimetext.io/language_servers/#sorbet)
```

Comment:
:wave: Would it be feasible to open on a PR `submlimelsp` to add built-in support for Ruby LSP, rather than having people manually configure it?